setup.py

```diff
@@ -19,6 +19,22 @@
     long_description = f.read()
 
 
+def format_installs_required(config):
+    dependencies = []
+    for k, v in config.items():
+        if k == "python":
+            continue
+        elif k in ["werkzeug", "requests"]:
```

Comment:
Do we just want the condition `elif "version" in v and "markers" in v`? Rather than checking for specific packages?
Where is the format for the dependencies documented?